govuk-social-providers-only/login/login.ftl

```diff
@@ -75,5 +69,12 @@
                 </ul>
             </div>
         </#if>
+
+        <#if realm.password && realm.registrationAllowed && !usernameEditDisabled??>
+            <div id="kc-registration">
+                <h2 class="heading-medium">${msg("noAccount")}</h2>
+                <span><a href="${url.registrationUrl}">${msg("registerLink")}</a></span>
+            </div>
+        </#if>
```

Comment:
Looks great. Two tiny queries:

1. Might it be worth keeping it in the same place in the file as it was before?
2. Maybe we wrap the link in a `<p>` rather than a `<span>`, or just remove the span (as I don’t think we need it).